src/test/java/com/revature/maadcars/controllers/UserControllerTest.java

```diff
@@ -77,4 +77,15 @@ void saveUserToDatabase() throws Exception {
                 .andReturn();
     }
 
+    @Test
+    void saveUserToDatabaseExceptBadInput() throws Exception {
+        when(userService.saveUser(any(User.class))).thenThrow(IllegalArgumentException.class);
```

Comment:
Optional: later tweak your exception-catching tests to also attach a message to thrown exception and make sure the message is in the HTTP response body.